.github/workflows/backport.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   backport:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I am not sure about that, do we have any known issue regarding to ubuntu-latest. Any reason change to 20.04?
I think opensearch-project use ubuntu-latest by default.

Reply:
https://github.com/Bit-Quill/opensearch-project-sql/pull/118#discussion_r971385524
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
**TL;DR**
`ubuntu-latest` = `ubuntu-20.04` now, but subject to change